docs/operate/get-started/other-hardware/_index.md

```diff
@@ -987,16 +987,16 @@ Do not change the <code>module_id</code>.</p>
 </tr>
 <tr>
 <td><code>models</code></td>
-<td>object</td>
-<td><strong>Required</strong></td>
-<td><p>A list of one or more {{< glossary_tooltip term_id="model" text="models" >}} provided by your custom module. You must provide at least one model, which consists of an <code>api</code> and <code>model</code> key pair. If you are publishing a public module (<code>"visibility": "public"</code>), the namespace of your model must match the <a href="/operate/reference/naming-modules/#create-a-namespace-for-your-organization">namespace of your organization</a>.</p>
+<td>array</td>
+<td>Optional</td>
```

Comment:
```suggestion
<td><strong>Required</strong> if your module provides models</td>
```
[opt] Maybe better to emphasize like this since it's not just plain optional, and this is within the integrate hardware section?

Reply:
It makes that column too long. If anyone makes a mistake with this the CLI will tell them about it and mostly this is auto-generated, so I'm not worried that there will be confusion.